lambo

```diff
@@ -351,10 +350,15 @@ perlCommands=(
 )
 
 for perlCommand in "${perlCommands[@]}"; do
-    perl -pi -e "$perlCommand" ".env.example"
     perl -pi -e "$perlCommand" ".env"
 done
 
+cp .env .env.example
```

Comment:
I know it doesn't *really* matter, but I think I'd rather we do the modifications to the `.env.example` file and then copy it to `.env`. Technically it'll be exactly the same but it makes a little bit more sense to my brain :)